### PR TITLE
calceph 3.4.6

### PR DESCRIPTION
**Pull request by gastineau**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by SMillerDev — DISMISSED** [review]

**Comment by gastineau** [comment]
add there license and squash commits to obtain only one commit per pull-request

**Review by SMillerDev — APPROVED** [review]

File: Formula/calceph.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Calceph < Formula
   desc "C library to access the binary planetary ephemeris files"
   homepage "https://www.imcce.fr/inpop/calceph"
-  url "https://www.imcce.fr/content/medias/recherche/equipes/asd/calceph/calceph-3.4.5.tar.gz"
-  sha256 "e1b9076b49fee51f7efda22194c8b66aaeebedbdb41d731c5af2cd0b16b39b56"
+  url "https://www.imcce.fr/content/medias/recherche/equipes/asd/calceph/calceph-3.4.6.tar.gz"
+  sha256 "fc2c0d899c32ddee2d484159c6fda5ae1ee7d8beae517029b776e5250b5fda27"
+  license "GPL-2.0-or-later"
```
Comment (Rylan12) [inline Formula/calceph.rb:6]: @gastineau where do you see `GPL-2.0-or-later`?

From the homepage:

> This software is an open-source product distributed under three distinct licenses giving you the choice of one of the three following sets of free software/open source licensing terms. This allows the use of our code in as wide a variety of software projects as possible.
>
> This software is governed by the [CECILL-C](https://cecill.info/licences/Licence_CeCILL-C_V1-en.html), [CECILL-B](https://cecill.info/licences/Licence_CeCILL-B_V1-en.html) or [CECILL version 2.1](https://cecill.info/licences/Licence_CeCILL_V2.1-en.html) license under French law and abiding by the rules of distribution of free software. You can use, modify and/or redistribute the software under the terms of the CeCILL-C, CeCILL-B or CeCILL license as circulated by CEA, CNRS and INRIA at the following URL https://cecill.info/.

Comment (gastineau) [reply Formula/calceph.rb:6]: 'CECILL version 2.1'  is compatible with GPL. According to the Formula-Cookbook, the approved license are : 'We only accept formulae that use a Debian Free Software Guidelines license, ...' . As the list is very restrictive, the 'GPL' was chosen to remain compatible.

Comment (gromgit) [reply Formula/calceph.rb:6]: "compatible" != "is". There's only one connection between CECILL and GPL: if the user injects GPL components into CECILL-licensed software, or vice versa, the merged result can be _relicensed_ as GPL. This does not apply here, so calceph remains licensed under CECILL.

**Pull request closed without merging**